lib/pluginmanager/package.rb

```diff
@@ -0,0 +1,67 @@
+class LogStash::PluginManager::Package < LogStash::PluginManager::Command
+
+  option "--tgz", :flag, "compress package as a tar.gz file", :default => false
+  option "--zip", :flag, "compress package as a zip file", :default => false
```

Comment:
I think I would pick one as default depending on the platform? on non-windows I'd pick tgz as the default ?